interfaces/seccomp/template.go

```diff
@@ -549,3 +549,23 @@ pwritev
 # of socket(), bind(), connect(), etc individually.
 socketcall
 `)
+
+// bindSyscallWorkaround specifies the bind() syscall for systems which
+// don't use AppArmor but only seccomp. In those cases the net package
```

Comment:
@niemeyer - your text is indeed better.

As for only supporting both enabled or both disabled, this was always the plan until more recently when people argued that there may be some security benefit to enabling seccomp without apparmor. I agree that while the seccomp-only mediation is not complete by any means, there is security benefit to having only it enabled. We discussed how bind() may not be the only syscall that is problematic but I feel that there shouldn't be too many of these based on the history of the snappy security policy. I'm fine with going back to the original 'all or nothing' design, but assuming people still want seccomp-only support, I think a wait and see approach is ok; we can always revisit this if/when needed.